templates/authentication-typescript/src/actions/RegistrationUIActions.tsx

```diff
@@ -58,14 +58,6 @@ export const ProjectRegistrationUIActions: () => RegistrationUIActions = () => (
         }
     },
 
-    /**
```

Comment:
Any particular reason for removing the comments that explain what the actions need to do?